src/lib/layout/Searchbar.tsx

```diff
@@ -301,7 +329,13 @@ const Searchbar = () => {
         onClose();
       }
     },
-    [keyword, metadata.icns.address, navigate, onClose]
+    [
+      keyword,
+      metadata.icns.address,
+      metadata.initiaUsername.address,
+      navigate,
+      onClose,
+    ]
   );
```

Comment:
IMO, inline condition that contain more than 2 conditions is a little bit hard to read (line: `160`, `321`). Shall we separate it into function?